torch/testing/_internal/logging_tensor.py

```diff
@@ -53,7 +53,7 @@ def __new__(cls, elem, *args, **kwargs):
         return r
 
     def __repr__(self):
-        return f"LoggingTensor({self.elem})"
+        return super().__repr__(tensor_contents=f"{self.elem}")
```

Comment:
Possible approach to defer most of the representation handling to the base `Tensor.__repr__()` while customizing the "tensor contents" portion. The other parts include autograd info, metadata, parameter-ness, tangent (if dual), etc.

Reply:
Yeah. We can add some optional arguments to `__repr__` to make this easier. Let's do it.